service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingVenueLookUpService.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.HearingVenue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+@Service
+public class HearingVenueLookUpService {
+
+    private final ObjectMapper objectMapper;
+    private final ResourceLoader resourceLoader;
+
+    public HearingVenueLookUpService(ObjectMapper objectMapper, ResourceLoader resourceLoader) {
+        this.objectMapper = objectMapper;
+        this.resourceLoader = resourceLoader;
+    }
+
+    private List<HearingVenue> getHearingVenueMappings() {
+
+        try {
+            File hearingVenueJsonFile = resourceLoader.getResource(
+                "classpath:static_data/hearingVenues.json").getFile();
+            String json = Files.readString(hearingVenueJsonFile.toPath());
+            return objectMapper.reader()
+                .forType(new TypeReference<List<HearingVenue>>() {})
+                .readValue(json);
+
+        } catch (IOException e) {
+            //should we stop processing ???
```

Comment:
The benefit of the above is no checked exception is thrown which anyway shows programmatic error.